dash/include/dash/util/UnitLocality.h

```diff
@@ -63,12 +63,12 @@ class UnitLocality
   }
 
   UnitLocality(
-    dart_unit_t   unit)
-  : UnitLocality(dash::Team::All(), unit)
+    global_unit_t   unit)
+  : UnitLocality(dash::Team::All(), local_unit_t(unit))
   { }
```

Comment:
Sorry, got your questions wrong: dash::Team::All() is a team like any other, so we have to pass a team-local unit ID to this constructor. We cannot handle the difference between `dash::Team::All()` and arbitrary teams with the types we have (unfortunately).